server/routes/user.py

```
@@ -0,0 +1,241 @@
+        return (
+            jsonify(
+                {
+                    "status": "error",
+                    "message": str(e),
+                    "data": None,
+                }
+            ),
+            400,
+        )
+
+    except IntegrityError:
```

Ruff (0.14.3)

try-except block with duplicate exception `IntegrityError`

(B025)